assets/javascripts/modules/SSO_links.js

```diff
@@ -36,7 +36,14 @@ module.exports = function(element, ssoUrl, ssoMethod) {
 
     if (clientSso || serverSso) {
 
-      elementHref = element.href;
+      destinationUrl = element.formaction || element.href;
+
+      //if destinationUrl is still not defined then the elemnt should be a button and we are using a non HTML5 compliant browser that cannot access the element.formaction attribute
+      if (!destinationUrl) {
+        var buttonParentForm = $element.closest('form')[0];
+        destinationUrl = (buttonParentForm) ? buttonParentForm.action : '';
+      }
+
```

Comment:
With this we can also just accept if a developer hasn't put the button inside a form then the `destinationUrl` will be `undefined` or at the worst if the button is not in a form but there is a form on the page then it will send through the action from that form. The developer will then release there is something wrong and then place the `button` in the `form`